src/app/shared/components/thumbnail/thumbnail.component.ts

```diff
@@ -39,19 +41,23 @@ export class ThumbnailComponent
   private imageElement: Element;
   private resizableImageElement: Element;
 
+  private INITIAL_ZOOM_LEVEL = 0.2963835165168812;
```

Comment:
Does this variable end up getting used at all? If not can we delete it?

Reply:
that is the initial zoom level which i forgot about, sorry! I can always check if the zoom level is equal to the initial one then we can swipe the image, otherwise we can pan, what do you think?